vm/vm.go

```diff
@@ -165,12 +162,13 @@ func New(
 	}
 
 	return &VM{
+		Registry: *NewRegistry(
+			registry.ActionRegistry(),
+			registry.AuthRegistry(),
+			registry.OutputRegistry()),
```

Comment:
Why create a registry from the existing registry? Can we just set the registry to the `chain.Registry` value that was passed in?

Reply:
done